zenoss.py

```diff
@@ -81,13 +81,13 @@ def get_rrd_values(self, device, dsnames, start=None, end=None, function='LAST')
         params = {'dsnames': dsnames, 'start': start, 'end': end, 'function': function}
         return ast.literal_eval(self.__session.get(url, params=params).content)
 
-    def get_devices(self, device_class='/zport/dmd/Devices', limit=None):
+    def get_devices(self, device_class='/zport/dmd/Devices', limit=None, **kwargs):
         '''Get a list of all devices.
 
```

Comment:
@coryschwartz  thanks for the addition. Could you add some comments here to describe what it does with an example or two?